### ISSUE TITLE
Document is broken on https://www.openssl.org/docs/manmaster/

### ISSUE THREAD

**Issue by eaufavor** [openssl/web#197]
https://www.openssl.org/docs/manmaster/ leads to a broken page at the moment. 

```
  | Name | Last modified | Size | Description
-- | -- | -- | -- | --

  | Parent Directory |   | - |  
  | man1/ | 2020-09-26 12:05 | - |  
  | man3/ | 2020-09-26 12:05 | - |  
  | man5/ | 2020-09-26 12:05 | - |  
  | man7/ | 2020-09-26 12:05 | - |  

```

Links like https://www.openssl.org/docs/manmaster/man3/SSL_read.html leads to 
```
Page Not Found
Sorry, but the link you gave does not exist.

To get the latest news about OpenSSL, download the source, and so on, please see the sidebar or the buttons at the top of every page. For more information about the team and community around the project, or to start making your own contributions, start with the community page.

You are here: Home
Sitemap
```

https://www.openssl.org/docs/man1.1.1/ works fine.

**Comment by mspncp** [comment]
Thanks for the heads-up @eaufavor. This issue was also noticed by @romen, but reported only internally. We will look into it, but it might take a little while.

**Comment by levitte** [comment]
I've looked at what happens, and it turns out that there's a fault in the OpenSSL Configure script, that has it fail with certain very basic config targets...

**Comment by levitte** [comment]
Fix in openssl/openssl#13031

**Comment by levitte** [comment]
openssl/openssl#13031 is merged, and I've done the needed maintenance.  Master manpages are back up.
